shrI-sampradAyaH/kriyA/vaikuNTha-dIxita-prapanna-dharma-sAra-samuchchayaH.md

```diff
@@ -7,11 +7,13 @@ Source: [TW](https://archive.org/details/in.ernet.dli.2015.495556)
 
 TODO: परिष्करणीयम्
 
-भूमिका, 
+श्रीः.
+
+भूमिका. 
 
 प्रपन्न धर्मसार समुच्चयम् 
 
-इदँकिलान्वर्थ नामधे यं प्रबद्धरत्नं श्रुत पूर्वम व्यदृष्ट चरवि-ति नपुनरावेदनी य मुत्तमग्रस्थ पर्यवेक्षण समुत्कना मा स्तीकशिरोमणी नाम्. तदिद मिदानीं दैवा दधिगत मस्माभिः* 
+इदंकिलान्वर्थ नामधेयं प्रबन्धरत्नं श्रुत पूर्वमप्यदृष्टचरमिति नपुनरावेदनीयमुत्तमग्रन्थ पर्यवेक्षण समुत्कण्ठानामास्तिकशिरोमणीनाम्. तदिदमिदानीं दैवादधिगतमस्माभिः. 
```

Comment:
Are you familiar enough with sanskrit to recognize long samAsa-s and upapada-s within them? If so, you could join them with - signs rather than leaving whitespaces.

Reply:
I am aware to an extent on recognizing the long samasas.  I present my understanding here w.r.t usage of whitespaces.
(i) after a vibakthi pratyaya 
(ii) after a tinganta
essentially (i) and (ii) indicating 1.4.14
(iii) after an avyaya
Whitespace appearing anywhere else (other than formatting) can be considered erroneous and needs correction.
This is in theory, how good my eye is with catching the unintended whitespaces; only can be found out on further corrections.
इदंकिलान्वर्थ नामधेयं प्रबन्धरत्नं श्रुत पूर्वमप्यदृष्टचरमिति नपुनरावेदनीयमुत्तमग्रन्थ पर्यवेक्षण समुत्कण्ठानामास्तिकशिरोमणीनाम्. तदिदमिदानीं दैवादधिगतमस्माभिः.
would be more correct as
इदंकिलान्वर्थनामधेयं प्रबन्धरत्नं श्रुतपूर्वमप्यदृष्टचरमिति नपुनरावेदनीयमुत्तमग्रन्थपर्यवेक्षणसमुत्कण्ठानामास्तिकशिरोमणीनाम्। तदिदमिदानीं दैवादधिगतमस्माभिः।
I am not so clear on the need to hyphenate upapadas within the samasthapadas.
(i) With lots of hyphens within the samasthapada the text might be rendered slightly unreadable to the reader.
(ii) If there is a sandhi operated between two upapadas within a samasthapada, I may have resort to padavichcheda only to insert the hyphen.
I should be able to do the first part with some guidance from you in the intial stages.

On a different note, I see there is an excessing use of anusvara within a padam.  Eg., इदंकिलान्वर्थ may more appropriately be written as इदङ्किलान्वर्थ.

Reply:
नमस्ते। इदंकिलान्वर्थनामधेयं is actually missing a whitespace. should be इदं किलान्वर्थनामधेयं . But no worries -  given the lack of expertise, forget about whitespaces. Just having the letters exactly as printed would be a great contribution

Reply:
धन्यवादः।  I overlooked the first word.  what about the rest in, इदंकिलान्वर्थनामधेयं प्रबन्धरत्नं श्रुतपूर्वमप्यदृष्टचरमिति नपुनरावेदनीयमुत्तमग्रन्थपर्यवेक्षणसमुत्कण्ठानामास्तिकशिरोमणीनाम्। तदिदमिदानीं दैवादधिगतमस्माभिः।, are they properly white-spaced?

Reply:
> इदं किलान्वर्थनामधेयं प्रबन्धरत्नं श्रुतपूर्वमप्यदृष्टचरमिति नपुनरावेदनीयमुत्तमग्रन्थपर्यवेक्षणसमुत्कण्ठानामास्तिकशिरोमणीनाम्। तदिदमिदानीं दैवादधिगतमस्माभिः।

is fine. My personal preference (for parsing facility would be note new lines, and imagine speaking it aloud with such pauses) 

> इदं किलान्वर्थनामधेयं प्रबन्धरत्नं  
श्रुतपूर्वम् अप्यदृष्टचरम्  
इति नपुनर् आवेदनीयम्  
उत्तमग्रन्थपर्यवेक्षणसमुत्कण्ठानाम् आस्तिकशिरोमणीनाम्। 
तद् इदम् इदानीं दैवाद् अधिगतम् अस्माभिः।

(In addition, I tend to hyphenate a lot while reading, for example https://github.com/vishvAsa/AgamaH_vaiShNavaH/tree/content/shrI-sampradAyaH/tattvam/parichaya-sanxepAH/vedAnta-kArikAvalI/sarva-prastutiH )